tests/playwright/src/ai-lab-extension.spec.ts

```diff
@@ -145,6 +146,51 @@ test.describe.serial(`AI Lab extension installation and verification`, { tag: '@
     });
   });
 
+  ['ggerganov/whisper.cpp'].forEach(modelName => {
+    test.describe.serial(`Model service creation and deletion`, () => {
+      let catalogPage: AILabCatalogPage;
+      let modelServiceDetailsPage: AILabServiceDetailsPage;
+
+      test.skip(isLinux, `Skipping model service creation on Linux`);
```

Comment:
Why Linux is skipped ?

Reply:
@jeffmaury deployment of service model/AI lab apps does not work on linux cicd